tools/pipelines/build-client.yml

```diff
@@ -60,20 +60,16 @@ trigger:
     - release/*
   paths:
     include:
-    - .prettierignore
-    - biome.json
-    - biome.jsonc
-    - packages
+    # Trigger on any edits to files in the root
+    - "*"
     - azure
+    - common/build/build-common
     - examples
     - experimental
-    - common/build/build-common
-    - lerna.json
-    - package.json
-    - pnpm-lock.yaml
-    - pnpm-workspace.yaml
+    - packages
+    - scripts
     # markdown-magic is part of the client release group
-    - tools/markdown-magic/*
+    - tools/markdown-magic
```

Comment:
I can't confirm, but the other syntax here seems to indicate that paths to folders will trigger on any edit in the folder.